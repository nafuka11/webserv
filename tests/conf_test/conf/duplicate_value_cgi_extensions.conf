server{
    location /cgi/ {
        cgi_extension php python;
        cgi_extension php;
    }
}