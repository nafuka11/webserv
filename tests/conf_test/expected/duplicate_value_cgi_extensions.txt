webserv: duplicate "cgi_extension:php" value. This value is already set. in ./conf/duplicate_value_cgi_extensions.conf:4
